bots/wallet/bot.js

```diff
@@ -0,0 +1,183 @@
+        };
+    }
+    if (!params.amount) {
+        return {
+            errors: [
+                status.components.validationMessage(
+                    I18n.t('validation_title'),
+                    I18n.t('validation_amount_specified')
+                )
+            ]
+        };
+    }
+
+    try {
+        var val = web3.toWei(params.amount.replace(",", "."), "ether");
+        if (val <= 0) { throw new Error(); }
+    } catch (err) {
+        return {
+            errors: [
+                status.components.validationMessage(
+                    I18n.t('validation_title'),
+                    I18n.t('validation_invalid_number')
+                )
+            ]
+        };
+    }
+
+    var balance = web3.eth.getBalance(context.from);
+    var estimatedGas = web3.eth.estimateGas({
+        from: context.from,
+        to: context.to,
+        value: val
+    });
+    if (bn(val).plus(bn(estimatedGas)).greaterThan(bn(balance))) {
+        return {
+            errors: [
+                status.components.validationMessage(
+                    I18n.t('validation_title'),
+                    I18n.t('validation_insufficient_amount')
+                    + web3.fromWei(balance, "ether")
+                    + " ETH)"
+                )
+            ]
+        };
+    }
+}
+
+function sendTransaction(params, context) {
+    var data = {
+        from: context.from,
+        to: context.to,
+        value: web3.toWei(params.amount.replace(",", "."), "ether")
+    };
+
+    try {
+        return web3.eth.sendTransaction(data);
+    } catch (err) {
+        return {error: err};
+    }
+}
+
+var send = {
+    name: "send",
+    icon: "money_white",
+    color: "#5fc48d",
+    title: I18n.t('send_title'),
+    description: I18n.t('send_description'),
+    params: [{
+        name: "amount",
+        type: status.types.NUMBER
+    }],
+    preview: function (params, context) {
+        var amountStyle = {
+            fontSize: 36,
+            color: "#000000",
+            height: 40
+        };
+
+        var amount = status.components.view(
+            {
+                flexDirection: "column",
+                alignItems: "flex-end",
+            },
+            [status.components.text(
+                {
+                    style: amountStyle,
+                    font: "light"
+                },
+                status.localizeNumber(params.amount, context.delimiter, context.separator)
+            )]);
+
+        var currency = status.components.view(
+            {
+                style: {
+                    flexDirection: "column",
+                    justifyContent: "flex-end",
+                    paddingBottom: 0
+                }
+            },
+            [status.components.text(
+                {
+                    style: {
+                        color: "#9199a0",
+                        fontSize: 16,
+                        lineHeight: 18,
+                        marginLeft: 7.5
+                    }
```

Comment:
I wanted to outline the general idea (don't put everything in a single file :) ), details can be different for bots